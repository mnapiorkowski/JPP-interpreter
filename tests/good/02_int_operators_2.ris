void main():
    let int a = 42;
    let int b = 10;
    
    println -a;
    println a + b;
    println a - b;
    println a * b;
    println a / b;
    println a % b;
    println a + b * a;
    
    println a != b;
    println a == b;
    println a > b;
    println a >= b;
    println a < b;
    println a <= b;

    return;