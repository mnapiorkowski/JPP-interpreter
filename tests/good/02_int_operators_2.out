-42
52
32
420
4
2
462
true
false
true
true
false
false
